include/deal.II/grid/reference_cell.h

```diff
@@ -681,6 +681,18 @@ class ReferenceCell
   double
   volume() const;
 
+  /**
+   * Return the $d - 1$-dimensional measure of the face of a reference cell that
+   * corresponds to the current object, where $d$ is the dimension of the space
+   * it lives in.
+   *
+   * @note Like ReferenceCell::volume(), this function defines vertices to have
+   * a measure of 1 to enable the definition of one-dimensional face
+   * quadratures.
+   */
+  double
+  face_measure(const unsigned int face_no) const;
```

Comment:
Isn't this equivalent to just saying `cell.face(face_no).volume()`?

Reply:
Unfortunately no. For instance, since the reference triangle is not an equilateral triangle its faces all have different measures. Similarly the base of the reference pyramid has an area of 4.

Reply:
Oh, good point. Since that thought had evaded me, maybe add something to this end to the documentation of this function?

Reply:
Good idea!